pkg/ddevapp/ddevapp.go

```diff
@@ -887,13 +887,18 @@ func (app *DdevApp) Down(removeData bool) error {
 		return fmt.Errorf("failed to remove %s: %v", app.GetName(), err)
 	}
 
-	// Remove data/database if we need to.
+	// Remove data/database/hostname if we need to.
 	if removeData {
```

Comment:
I think it's worth just a small conversation on whether we should just do this on regular rm as well. I guess the argument against is that people hit sudo too often.